testing.py

```diff
@@ -0,0 +1,78 @@
+import praw
+
+class RedditConfig:
+    id = '3iaAttC7v0D96Q'
+    secret = 'Dm-EfQSVyl8skVenes3DH25QfkY'
```

Comment:
Should these credentials be on GitHub?

Reply:
Hi @42B,

Thanks for your interest in our project. You are absolutely correct that these credentials should not be on GitHub, thank you for pointing that out. I have removed the keys so they should no longer be valid and will generate new ones which we will handle better.

Reply:
Also @42B, could I ask how you found our project?

Reply:
@Brojowski 

Sure, I go to this [url](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+language%3Apython+) and look at new pull requests in Python for practice.